File: src/main/java/rbq/wtf/lycoris/client/wrapper/wrappers/annotation/repeat/WrapConstructors.java
package rbq.wtf.lycoris.client.wrapper.wrappers.annotation.repeat;

import rbq.wtf.lycoris.client.wrapper.wrappers.annotation.WrapConstructor;

import java.lang.annotation.*;

@Documented
@Target(ElementType.FIELD)
@Retention(RetentionPolicy.RUNTIME)
public @interface WrapConstructors {
    WrapConstructor[] value();
}

File: src/main/java/rbq/wtf/lycoris/client/wrapper/wrappers/annotation/repeat/WrapConstructors.kt
package rbq.wtf.lycoris.client.wrapper.wrappers.annotation.repeat

import rbq.wtf.lycoris.client.wrapper.wrappers.annotation.WrapConstructor

@MustBeDocumented
@Target(AnnotationTarget.FIELD)
@Retention(AnnotationRetention.RUNTIME)
annotation class WrapConstructors(vararg val value: WrapConstructor)